openstates/la/__init__.py

```diff
@@ -20,13 +20,14 @@
                 ]
         },
         {
-            "name": "2012-2015",
+            "name": "2012-2016",
             "start_year": 2012,
-            "end_year": 2015,
+            "end_year": 2016,
             "sessions": [
                 "2012",
                 "2013",
                 "2014",
+		"2015",
```

Comment:
```diff
+		"2015",
```

PEP8 convention is to use spaces -- not tabs

Could you change all the tabs to four spaces? I'm a bit suprised this runs, actually!